OpenRA.Mods.Common/Traits/Attack/AttackBase.cs

```diff
@@ -31,8 +31,8 @@ public abstract class AttackBaseInfo : PausableConditionalTraitInfo
 		[Desc("Does the attack type require the attacker to enter the target's cell?")]
 		public readonly bool AttackRequiresEnteringCell = false;
 
-		[Desc("Does not care about shroud or fog. Enables the actor to launch an attack against a target even if he has no visibility of it.")]
-		public readonly bool IgnoresVisibility = false;
+		[Desc("Allow firing into the fog to target frozen actors without requiring force-fire.")]
+		public readonly bool TargetFrozenActors = false;
```

Comment:
Are there examples where Actors have longer firing range than vision where we would disable this? Or in other words, can we take the burden of remembering having to set this from the modder and automatically detemine it via code instead?

Reply:
Unfortunately yes, several.  RA infantry set a weapon range ~1 cell larger than their range, so depending on their subcell and angle of attack they can sporadically get hung-up on frozen dead buildings if assault-moved into an enemy base.